src/vivarium_census_prl_synth_pop/constants/data_values.py

```diff
@@ -29,6 +29,8 @@
     4: "Military",
     5: "Other non-institutional",
 }
+HOUSING_TYPES = ["Standard"] + list(GQ_HOUSING_TYPE_MAP.values())
```

Comment:
This makes me a little nervous because it depends on dictionary order, but I guess that is okay now that we only support versions above 3.7?

Reply:
We don't happen to care about ordering anywhere we're using this, but you're right that if we were to care about ordering later and this was a random ordering it could lead to confusing bugs later. But yeah, dict ordering is deterministic as of 3.7 as you say so it doesn't really matter.